third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -1523,6 +1523,76 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("correction"),
       py::arg("keepdim") = false,
       py::return_value_policy::reference);
+  nvf_ops.def(
+      "uniform",
+      [](nvfuser::FusionDefinition::Operators& self,
+         nvfuser::Scalar minval,
+         nvfuser::Scalar maxval,
+         std::vector<nvfuser::Scalar>& shape,
```

Comment:
Note that this one is a little funny and doesn't work look good with the API refactor we had from #2451 

So with dynamic vs static shape, we are currently looking at `Scalar shape` vs `std::vector<nvfuser::Scalar>`, which suggests that we are in the hell with pybind11 overload again... I need to revisit the refactor in #2450 and #2451